File: src/sdk-reference/js/6/collection/create/snippets/create.js
kuzzle
  .collection
  .create('nyc-open-data', 'yellow-taxi', {
    properties: {
      license: { type: 'keyword' },
      driver: {
        properties: {
          name: { type: 'keyword' },
          curriculum: { type: 'text' }
        }
      }
    }
  })
  .then(() => {
    console.log('Success');
  })
  .catch(error => {
    console.error(error.message);
  });

File: src/sdk-reference/cpp/1/document/create/snippets/create.cpp
try {
  std::string response;
  response = kuzzle->document->create(
    "nyc-open-data",
    "yellow-taxi",
    "some-id",
    R"({"lastName": "Eggins"})"
  );

  std::cout << response << std::endl;
  /*
  {
    "_index": "nyc-open-data",
    "_type": "yellow-taxi",
    "_id": "some-id",
    "_version": 1,
    "result": "created",
    "_shards": {
      "total": 2,
      "successful": 1,
      "failed": 0
    },
    "created": true,
    "_source": {
      "lastName": "Eggins",
      "_kuzzle_info": {
        "author": "-1",
        "createdAt": 1537445737667,
        "updatedAt": null,
        "updater": null,
        "active": true,
        "deletedAt": null
      }
    }
  }
  */

  std::cout << "Document successfully created" << std::endl;
} catch (kuzzleio::KuzzleException& e) {
  std::cerr << e.what() << std::endl;
}
